http/filestream/filestream.go

```diff
@@ -27,7 +27,7 @@ func ReadFilesFromStream(multipartReader *multipart.Reader, fileWriterFunc FileW
 			}
 			return fmt.Errorf("failed to read file: %w", err)
 		}
-		err = readFile(fileReader, fileWriterFunc)
+		err = readFile(fileReader, fileWritersFunc)
 		if err != nil {
 			return err
 		}
```

Comment:
Preferebly, let's reduce scope of variables when possible
```suggestion
		if err = readFile(fileReader, fileWritersFunc); err != nil {
            return err
        }
```